### DISCUSSION TITLE
LOGIN_REQUIRED set to False, but content is still blocked for anybody

### DISCUSSION THREAD

**Discussion by mlazzarotto** [netbox-community/netbox-docker#787] [Getting Help] [answered]
Hi, I have a fresh new Netbox 3.2.5 install with LDAP authentication on docker-compose.
I want anybody to read Netbox data, but if I open Netbox without logging in, I see all the menu items locked with lock icon

I also tried to set LOGIN_REQUIRED to False, but that didn't work (I eventually found out that that's the default)
![image](https://user-images.githubusercontent.com/8932945/177559395-46f0f5f9-14b4-47d7-a6ff-2105b4f6d689.png)

**Comment by hydezhao** [comment, marked as answer]
you can try 
```
EXEMPT_VIEW_PERMISSIONS = ['*']
```
in `configuration/extra.py`
